kubevirt-101/step2.md

```diff
@@ -1,74 +1,60 @@
-#### Deploy a VM
+# Deploy a Virtual Machine
 
-The command below applies a YAML definition of a virtual machine into our current Kubernetes environment, defining the VM name, the resources required (disk, CPU, memory), etc. You can take a look at the [vm.yaml](https://kubevirt.io/labs/manifests/vm.yaml) file if you have interest in knowing more about a virtual machine definition:
+The command below applies a YAML definition of a virtual machine into the current Kubernetes environment, defining the VM name, the resources required (disk, CPU, memory), etc. You can take a look at the [vm.yaml](https://kubevirt.io/labs/manifests/vm.yaml) file if you have interest in knowing more about a virtual machine definition:
 
 `kubectl apply -f https://kubevirt.io/labs/manifests/vm.yaml`{{execute}}
```

Comment:
what are the {{execute}} tags I keep seeing. They render on output. Its not just this file I see them on.